### DISCUSSION TITLE
sdlf-cicd

### DISCUSSION THREAD

**Discussion by prajeshrawat** [aws-solutions-library-samples/data-lakes-on-aws#116] [Q&A]
HI SDLF,
I want to modify the code for uploading the artifacts/code-commit repo for dataLakeLibrary.
Where can I find the code/make the changes.
Thanks.

**Comment by malachi-constant** [comment]
Layer is created [here](https://github.com/awslabs/aws-serverless-data-lake-framework/blob/5fd31738332da359149891de723570dc6206e80e/sdlf-team/nested-stacks/template-cicd.yaml#L166) in cloudformation. Starting point source code for the layer is https://github.com/awslabs/aws-serverless-data-lake-framework/tree/main/sdlf-datalakeLibrary

**Reply by prajeshrawat** [reply]
@malachi-constant , the codebuild code mentioned about packages the layer and deploys it.But I am still not able to find  the code where  this repository is being written to s3 in cfn-artifacts bucket.

**Reply by malachi-constant** [reply]
[Codebuild job](https://github.com/awslabs/aws-serverless-data-lake-framework/blob/5fd31738332da359149891de723570dc6206e80e/sdlf-cicd/template-cicd-team-repos.yaml#L149) is created to bootstrap the team repos. Here is the [script and function](https://github.com/awslabs/aws-serverless-data-lake-framework/blob/5fd31738332da359149891de723570dc6206e80e/sdlf-team/scripts/bootstrap_team.sh#L19) called specifically where repo is created and local files are pushed to remote.

**Reply by prajeshrawat** [reply]
Thanks @malachi-constant
